webapp/crashstats/libbugzilla.py

```diff
@@ -134,10 +134,12 @@ def crash_report_to_description(crash_report_url, processed_crash):
     ]
     if processed_crash.get("moz_crash_reason"):
         lines.append("")
-        lines.append(f"MOZ_CRASH Reason: ```{processed_crash['moz_crash_reason']}```")
+        lines.append(
+            f"MOZ_CRASH Reason:\n```\n{processed_crash['moz_crash_reason'].strip()}\n```\n"
```

Comment:
The `lines` array is joined using `\n` as a separator, so I probably would have done this like we do the stack below:

```
lines.append("```")
lines.append(processed_crash("moz_crash_reason").strip())
lines.append("```")
```